ansible-scylla-node/defaults/main.yml

```diff
@@ -201,6 +201,14 @@ scylla_io_probe: True
 # and then the configuration files will be propagated for the other nodes of each datacenter.
 scylla_io_probe_dc_aware: False
 
+# If set to True, the role will always replace the io.conf file and you will
+# lose any manual values you might have configured on the nodes.
+always_replace_io_conf: False
+
+# If set to True, the role will always replace the io_properties.yaml file and you will
+# lose any manual values you might have configured on the nodes.
```

Comment:
One way to describe it would be as follows:
"If set to True the content of io_properties.yaml is going to aligned with the first node or the first node in each DC (depending on the value of scylla_io_probe_dc_aware)".

Reply:
Fixed